src/Framework/Dumper.php

```diff
@@ -326,7 +326,7 @@ public static function saveOutput($testFile, $content, $suffix = '')
 			$path = dirname($testFile) . DIRECTORY_SEPARATOR . $path;
 		}
 		@mkdir(dirname($path)); // @ - directory may already exist
-		file_put_contents($path, is_string($content) ? $content : self::toPhp($content));
+		file_put_contents($path, is_string($content) ? $content : self::toPhp($content) . "\n");
 		return $path;
```

Comment:
That's intentional, I didn't want to introduce newline to string `$content`. If that does not end with newline, diff tools should still show the original warning.